Makefile

```diff
@@ -382,6 +382,17 @@ else ifeq ($(JULIA_BUILD_MODE),debug)
 endif
 endif
 
+	# Fix rpaths for dependencies. This should be fixed in BinaryBuilder later.
+ifeq ($(OS), Linux)
+	-$(PATCHELF) --set-rpath '$$ORIGIN' $(DESTDIR)$(private_shlibdir)/libgfortran.$(SHLIB_EXT)
```

Comment:
This one shouldn't be needed anymore after #47381 (now merged)

Reply:
Neat. What should we do about LLVM?

Reply:
I'd say keep the line below for the time being.  Investigating what the audit does in the build of llvm is way more complicated.